Makefile

```diff
@@ -4,6 +4,8 @@ CI_REGISTRY ?= docker.io
 IMAGE_PREFIX ?= $(CI_REGISTRY)/7val/
 CURRENT_BRANCH ?= $(shell git symbolic-ref --short HEAD)
 ONLY_BRANCH ?= master
+# needed to mount files inside a container started inside a container
+HOST_PATH=$(shell pwd)
```

Comment:
When using `.EXPORT_ALL_VARIABLES:` before the variables definitions then you could leave out the `@export HOST_PATH=$(HOST_PATH);` in so many places.